metabase/metabase
# Hide metabot

### Description

Hides Metabot experimental feature.

### How to verify

- Run an instance where Metabot has not been enabled
- Go to Admin
- Ensure there is no "Metabot" item in the sidebar
- Go to Metabot urls http://localhost:3000/metabot/database/1 and http://localhost:3000/metabot/model/1 and ensure they do not open
- Restart the app using `MB_IS_METABOT_ENABLED=true yarn dev`
- Ensure Metabot works as expected

### Checklist
